courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>Replace the title her by a markdown cell with something like "Let's now retrieve the training statistics."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>